src/libraries/Common/src/SkipLocalsInit.cs

```diff
@@ -1,5 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable disable
```

Comment:
Is it correct to disable nullability on files inside of the Common folder?

Reply:
I did it for now because other projects depend on it and enabling is causing errors which I do not want to fix as part of this PR. I'll open the issue on this

Reply:
https://github.com/dotnet/runtime/issues/41439